classes/PodsAPI.php

```diff
@@ -9390,9 +9390,7 @@ public function get_table_info( $object_type, $object, $name = null, $pod = null
 			$info['meta_field_index'] = 'meta_key';
 			$info['meta_field_value'] = 'meta_value';
 
-			$info['where'] = array(
-				'user_status' => '`t`.`user_status` = 0'
-			);
+			$info['where'] = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: db78b4a6c536a904c6f9655f989839e1</sub>